docs/data/README.mdx

```diff
@@ -70,3 +70,7 @@ Horizon is an API for accessing and interacting with the Stellar network data. I
 Horizon stores three types of data (current state, historical state, and derived state) in one database, and the data is available in real-time for transactional use, which makes Horizon more expensive and resource-intensive to operate. If you’re considering using Horizon over the RPC, let us know in the [Stellar Developer Discord](https://discord.gg/stellardev) or file an issue in the [RPC repo](https://github.com/stellar/soroban-rpc) and let us know why!
 
 You can [run your own instance of Horizon](./horizon/admin-guide/README.mdx) or use one of the publicly available Horizon services from [these infrastructure providers](./horizon/horizon-providers.mdx).
+
+## [Galexie](./galexie/README.mdx)
+
+Galexie is a tool for exporting Stellar ledger metadata to Google Cloud Storage.
```

Comment:
Are we saying `Google Cloud Storage` instead of Datastore/Data Lake/Galexsea because we only support GCS right now?